c7n/tags.py

```diff
@@ -1101,6 +1134,26 @@ def get_resource_tag_map(self, r_type, ids):
             for r in manager.get_resources(list(ids))
         }
 
+    def get_resource_tag_map_universal(self, ids):
+        related_region = None
+        ids = list(ids)
+        if ids:
+            if ids[0].startswith('arn:aws:iam'):
```

Comment:
this seems like a reasonable way to work around the fact that we can't check for the resource type's `global_resource` property. i see a few other global resource types (account, support, shield, route53), but as far as i can tell none of them are relevant for this action. guessing you went through the same review :) :+1: